ebml/EbmlElement.h

```diff
@@ -413,4 +427,6 @@ class EBML_DLL_API EbmlElement {
 
 } // namespace libebml
 
+#define EBML_WRITE_FILTER 1
```

Comment:
I… don't like this `#define`. Consumers can already use the version number `#defines` in order to differentiate between v1 & v2 of the library.

Not sure how I'll handle it in MKVToolNix. Will have to look into how many places will have to be changed as I'm generally not a fan of a lot of conditional compilation anyway. Maybe I'll just throw the switch & require v2 as soon as I add support for it.

Reply:
The define is there to tell v2 users which API they can expect. Once v2 is stable we can remove it and everyone should assume it's there for v2 code. In other words it's to make your life easier if you want to support the old and new API in your codebase for a while. I don't mind removing it and you handle it on your side.